+nosnoc/+dcs/Heaviside.m

```diff
@@ -6,16 +6,12 @@
         lambda_n_sys % cell
         lambda_p % CasADi symbolic variable 
         lambda_p_sys % cell
-        % These are relevant only for lifting. For now wait to implement until re-implementing time-freezing
-        % beta 
-        % gamma
-        % theta
-        % theta_sys
         theta_expr_sys 
```

Comment:
please a one line comment what is theta (cell of casadi expressions of alpha, defining theta..